Source/Scenes/Overworld.cs

```diff
@@ -174,6 +175,13 @@ public Overworld(bool startOnLastSelected)
 			new(new Vec3(-cardWidth, 0, cardHeight) / 2, new Vec2(0, 1), Color.White),
 		]);
 		mesh.SetIndices<int>([0, 1, 2, 0, 2, 3]);
+		
+		Game.OnResolutionChaned += () => mesh.SetVertices<SpriteVertex>([
+			new(new Vec3(-cardWidth, 0, -cardHeight) / 2, new Vec2(0, 0), Color.White),
+			new(new Vec3(cardWidth, 0, -cardHeight) / 2, new Vec2(1, 0), Color.White),
+			new(new Vec3(cardWidth, 0, cardHeight) / 2, new Vec2(1, 1), Color.White),
+			new(new Vec3(-cardWidth, 0, cardHeight) / 2, new Vec2(0, 1), Color.White),
```

Comment:
Do cardWidth and cardHeight need to be recalculated here too? I think these were only getting set in the constructor, but not updated when resolution changed. (talking specifically about the lowercase version, which seems to be different from the capitalized version.)